Specification Registry/Attributes/amount.json

```diff
@@ -0,0 +1,13 @@
+{
+    "fieldInternalName": "amount",
```

Comment:
What use case do you imagine for such a field?

Reply:
Was thinking about any number representing a monetary value.